### PR TITLE
Check if json content is valid

### PR DESCRIPTION
**Pull request by FlorentinGarnier**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?   | no
| Fixed tickets | #... 

Throw a new exception if content contain null by json_decode return

### CODE REVIEW

**Comment by FlorentinGarnier** [comment]
Hello,

Build has faile but it seems its from Sylius

From PHP Stan 
```
274    Call to an undefined method object::setPrice().                         
275    Call to an undefined method object::setOriginalPrice().                 
308    Parameter #1 $channel of method                                         

         Sylius\Component\Channel\Model\ChannelsAwareInterface::addChannel()     

         expects Sylius\Component\Channel\Model\ChannelInterface, object given.  
```

**Comment by FlorentinGarnier** [comment]
Normally it's done

File: src/Importer/JsonResourceImporter.php [hunk 1]
```diff
@@ -38,7 +38,13 @@ public function import(string $fileName): ImporterResultInterface
             throw new ImporterException(sprintf('File %s could not be loaded', $fileName));
         }
 
-        foreach (json_decode($contents, true) as $i => $row) {
+        $data = json_decode($contents, true);
+
+        if (is_null($data)) {
```
Comment (oallain) [inline src/Importer/JsonResourceImporter.php]: You need to write
```suggestion
        if (null === $data) {
```

for fix Ecs error in Travis.

**Comment by oallain** [comment]
Thanks @FlorentinGarnier 👍

**Pull request merged**